packages/devtools/lib/src/inspector/inspector_controller.dart

```diff
@@ -482,11 +482,10 @@ class InspectorController implements InspectorServiceClient {
       // Wait for the master to update.
       return;
     }
-    updateSelectionFromService();
+    updateSelectionFromService(false);
```

Comment:
Do we not have the unawaited future lint on anymore? Shouldn't it be showing a warning for this line?

Reply:
that lint only warns if the method calling the async method also happens to be async. in this case it isn't
btw in InspectorService unawaited futures are actually fine for the most part because the ObjectGroups are used as a hammer used to abandon obsolete work and wait for pending work to complete.